src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -216,12 +216,14 @@ export class FileShareDetailComponent implements OnInit{
             str.forEach(item=>{
                 if(item.fileshareId == fileShareId){
                     item.createdAt = Utils.formatDate(item.createdAt);
-                    if(!item.description){
-                        item.description = "--";
-                    }
+                    item.date = new Date(Utils.formatDate(item.createdAt)).getTime();
+                    item.description = (!item.description || item.description == '') ? '--' : item.description;
```

Comment:
please consider making a reusable util method out of this

Reply:
@rhsakarpos I have added this to the backlog ( #186 )  since this is something that will have to be implemented at the application level. If this is agreeable please consider resolving this comment.

Reply:
LG...AB